packages/next/src/build/templates/edge-ssr.ts

```diff
@@ -44,17 +41,8 @@ declare const user500RouteModuleOptions: any
 // INJECT:errorRouteModuleOptions
 // INJECT:user500RouteModuleOptions
 
-const cacheHandlers = {}
```

Comment:
If the cache handlers aren't injected here by our `loadEntrypoint` with the loader how are they injected into `initializeCacheHandlers`?

Reply:
The `initializeCacheHandlers()` call pulls from the symbols, so it should inject it the same way.

Reply:
As discussed in DM since this is edge specific and edge can't be configured currently for dynamic IO this isn't a blocker. We should prob add a comment so we remember this caveat in edge runtime if we do add support.